src/main/java/step4/model/LottoBuyCount.java

```diff
@@ -24,10 +24,36 @@ public LottoBuyCount(int count) {
         this.lottoBuyCount = count;
     }
 
+    public LottoBuyCount(String count) {
+        this(convertNumber(count));
+    }
+
+    private static int convertNumber(String text) {
+        int result;
+        try {
+            result = Integer.parseInt(text);
+        } catch (NumberFormatException e) {
+            throw new LottoFormatException(ErrorMessageConstant.NOT_NUMBER);
+        }
+        return result;
```

Comment:
지금은 원시타입이기 때문에 해당되지 않겠지만 
가능하다면 초기화가 안된 로컬변수는 NPE 가 발생될 수 있기 때문에 조심하는게 좋다고 생각하는데
혹시 이부분에 대해서 어떻게 생각하시나요?? 🤔
```suggestion
        try {
            return Integer.parseInt(text);
        } catch (NumberFormatException e) {
            throw new LottoFormatException(ErrorMessageConstant.NOT_NUMBER);
        }
```

Reply:
말씀해주신대로 다른 곳에서 이렇게 할 경우 NPE 발생가능성이 있겠네요 null 체크 로직을 추가하겠습니다.